internal/controller/standalone_pgadmin/pod.go

```diff
@@ -34,52 +46,77 @@ func pod(
 ) {
 	const (
 		// config and data volume names
-		configVolumeName = "standalone-pgadmin-config"
-		dataVolumeName   = "standalone-pgadmin-data"
+		configVolumeName = "pgadmin-config"
+		dataVolumeName   = "pgadmin-data"
+		logVolumeName    = "pgadmin-log"
+		scriptVolumeName = "config-system"
```

Comment:
👍🏻 I like `script` and `config-system`.

⛏️ I have some mild anxiety that something else might try to inject a (generic sounding) `config-system` volume. I would feel more comfortable if this had `pgadmin` on the front like the other volume names.

```suggestion
		scriptVolumeName = "pgadmin-config-system"
```

Reply:
Updated.